app/assets/stylesheets/dashboard.scss

```diff
@@ -469,6 +474,21 @@
         vertical-align: 0.16em;
       }
 
+      .dashboard-follow {
+        background: $blue;
```

Comment:
Can you use the themeable shortcuts? So that people can customize this for the other themes.

It'd be great if you could have a look at how it looks with the other times and add colors for this.